msd/SpeakerDiarizer.py

```diff
@@ -11,11 +13,29 @@ def __init__(self,name_pipe):
         self.diarization = None
         self.der = None
         self.current_filename = None
+        self.audio_profiles = None
+        self.profil_paths = None
 
-    def apply_diarizer(self,file_name):
+    def apply_diarizer(self,file_name,profil_paths):
```

Comment:
met un argument par défaut ici, vu qu'on a dans la meme branche les 3 fichiers wav
Dans ta classe tu peux créer une variable testing_paths = ['data/profiles/Martin.wav', 'data/profiles/Yoann.wav', 'data/profiles/Loïc.wav'] et l'appeler ici 
```suggestion
    def apply_diarizer(self,file_name,profil_paths=testing_paths):
```